monitoring/install.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Setup Docker
+  hosts: all
+  become: true
+  become_user: root
+  become_method: sudo
+  gather_facts: false
+  tasks:
+    - name: Install packages
+      shell: |
+        apt update
+        apt install -y docker.io
+        apt install -y unzip
+    - name: Set up Docker Swarm
```

Comment:
never used docker swarm--what's the advantage of using it over straight compose?

Reply:
for this use not much since it’s just a single machine. we could probably simplify down to just compose and be fine, but this is mostly cobbled together from a bunch of copy pasting so there are questionable design decisions all over the place